ci/release-info-v2.py

```diff
@@ -0,0 +1,96 @@
+import json
+import os
+import re
+import sys
+from datetime import date
+from subprocess import Popen, PIPE
+
+def read_version(crate):
+    # Reads package version from workspace manifest file
+    channel = Popen(
+        ['cargo', 'read-manifest', '--manifest-path', './{}/Cargo.toml'.format(crate)],
+        stdout=PIPE
+    )
+
+    # parse json data
+    data = json.load(channel.stdout)
+    version = data['version']
+    return version
+
+def check_version(ver):
+    # Checks package version for matching with the current tag reference
+    if ref is not None and ref != 'refs/tags/v' + str(ver):
+        return 0
```

Comment:
Why use numbers here?, you could simply return a boolean with the whole expression in the if clause.

Reply:
This is forming a bit mask for checking the crates version consistency in a whole bundle.

Reply:
This is used here: `consistency = list(map(check_version, bundle))`. Which value is used in an `all` call. `all` already check boolean(y) values. So we could actually keep the bools.

Reply:
Isn't Python resolve 1 and 0 into booleans in condition checks? This used in a simple typecast in condition checks - if v (element in consistency) is 0 this means that this package doesn't compliant with tag and bundle version.